docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,31 +6,40 @@ Please include a summary of the change and which issue is fixed. Please also inc
 
 Please delete options that are not relevant.
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Documentation update
-- [ ] Code cleanliness or refactor
-- [ ] Dependency change
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Documentation update
+- Code cleanliness or refactor
+- Dependency change
 
 ## Screenshot
 
-Please provide if applicable.
+Please provide if applicable (browser for `frontend` or any visual way to show a `backend` change).
 
 # Testing
 
-Manual steps to test this functionality:
+Applicable for all code changes.
```

Comment:
I like how @MichaelCherry said "N/A (no code changes)" under Testing and Checklist so I made a note about these sections being required only for code changes.

Reply:
https://github.com/code-for-canada/project-thundercat/pull/71